### ISSUE TITLE
Shut down write access to OpenSpending

### ISSUE THREAD

**Issue by rufuspollock** [openspending/openspending#1483]
Two ways for users to add datasets are packager and data genus processor => Shut down packager and data genus processor.

## Acceptance

* [ ] No way to add new datasets (or update existing ones)
* [ ] Pages replaced with some notice pointing to the migration issue

## Tasks

...

**Comment by rufuspollock** [comment]
@sglavoie can you consult with @akariv on how we might do this and then get ready to implement this.

**Comment by sglavoie** [comment]
@rufuspollock, yes, no problem! I will reach out to @akariv in a short while today and keep you updated. :+1:

**Comment by sglavoie** [comment]
Here is the explanation @akariv shared with me:

> Easiest way would be to shut down the os-packager and os-data-importers containers in the kubernetes environment.
You can use the kubernetes scale command to set the amount of replicas to 0, thus effectively removing them from the deployment.
